README.md

```diff
@@ -72,3 +72,27 @@ REACT_APP_TORNJAK_MANAGER=true npm start
 ```
 
 In this view, there is an additional navigation bar tab titled "Manage Servers" where you may register Tornjak agents.  
+
+### Running Tornjak With Auth
+Once you have Auth instance running: 
+Example - Keycloak instance running locally or on the cloud 
+
+To run Keycloak locally, run:
+
+```docker run -p 8080:8080 -e KEYCLOAK_ADMIN=”username” -e KEYCLOAK_ADMIN_PASSWORD=”password” quay.io/keycloak/keycloak:19.0.1 start-dev
```

Comment:
for some reason this is not showing up, and is hard to make suggestion for the triple quotes; i think the command needs to be on separate line
```suggestion
\```
docker run -p 8080:8080 -e KEYCLOAK_ADMIN=”username” -e KEYCLOAK_ADMIN_PASSWORD=”password” quay.io/keycloak/keycloak:19.0.1 start-dev
```
```

Reply:
done